SECURITY-ENVIRONMENT.md

```
@@ -0,0 +1,461 @@
+./scripts/health-check-security.sh
+
+# Comprehensive health check with resource monitoring
+./scripts/health-check-security.sh --comprehensive
+
+# Generate detailed health report
+./scripts/health-check-security.sh --comprehensive --report
+```
+
+### 3. Access Security Services
+
+| Service | URL | Purpose | Credentials |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)